resources/runners/qemu.sh

```diff
@@ -1,3 +1,16 @@
+APPEND="root=/dev/sda ip=dhcp vga=0x344"
+while [ ! $# -eq 0 ]
+do
+    case "$1" in
+        --terminal | -t)
+            APPEND="$APPEND console=ttyAMA0,115200 console=ttyS0"
```

Comment:
Ok I see, in that case it's ok.
El 27/11/2015 08:51, "Tyler James Leonhardt" <notifications@github.com>
escribió:

> In resources/runners/qemu.sh
> <https://github.com/NodeOS/NodeOS/pull/188#discussion_r46023754>:
>
> > @@ -1,3 +1,16 @@
> > +APPEND="root=/dev/sda ip=dhcp vga=0x344"
> > +while [ ! $# -eq 0 ]
> > +do
> > +    case "$1" in
> > +        --terminal | -t)
> > +            APPEND="$APPEND console=ttyAMA0,115200 console=ttyS0"
>
> I got it from here:
> http://stackoverflow.com/questions/19565116/redirect-qemu-window-output-to-terminal-running-qemu#comment42884295_19569340
>
> Apparently some emulated machines use it. I can take it out if you think
> we should.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NodeOS/NodeOS/pull/188/files#r46023754>.
>